python/py-gpstime/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-gpstime
+version             0.3.2
+
+platforms           darwin
+supported_archs     noarch
+license             GPL-3
+maintainers         gmail.com:stefan.countryman
```

Comment:
Please add your GitHub username and possibly openmaintainer here too.

Reply:
Just to be clear, this means that you should change this to

```
maintainer     {gmail.com:stefan.countryman @stefco} openmaintainer
```

Reply:
Thanks! Changed, will push a new commit with all fixes after fixing livecheck (below).